### DISCUSSION TITLE
I want to renew my GH education pack but I'm doing a distance program and I have no idea what I'm supposed to take a picture of with my camera to prove it

### DISCUSSION THREAD

**Discussion by aseyedia** [community/community#136237] [GitHub Education]
### Hi everyone,

I have my student card, which, by the way, says "off campus graduate" on it, but then I have to answer this question:

![image](https://github.com/user-attachments/assets/30ecbe7c-06c0-4d83-a8c5-9f25368431ba)

and Github won't allow me to, i don't know, upload my unofficial transcript. I can only take a pic with my webcam. Okay, what am I supposed to take a picture of? it already won't accept my student ID, so what else? Do I print out my transcript and take a picture of that? Why can't I just upload a PDF?

**Comment by ulyssecoispellier** [comment]
Do you have, by any chance, a certification from your school saying you are part of a program? If yes I believe you just need to print it and then take a picture with it 😀

**Reply by aseyedia** [reply]
Like an unofficial transcript or something? I have tried taking a pic of my transcript on the screen with my webcam and obviously that looked bad. I just can't shake the feeling that GH is trying to discourage student users, or at least distance students? Why would they make us have to take a picture of a printed out copy of the document? I tried doing this on my phone too, btw, and the page would only allow me to use the selfie camera. Idk if that was my phone or the page but I also found that to be odd. I guess I can try printing something but that is a weird process.

**Reply by ulyssecoispellier** [reply]
I believe so yes

**Reply by chicouskiluiz** [reply]
@aseyedia I had the same experience as you when sending documents, using photos the way they are now is terrible. Furthermore, I sent an official document issued by my college stating that my teaching is distance learning, I was rejected in the same way.

**Comment by canbedir** [comment]
Did you figure it out?

**Comment by aseyedia** [comment]
No, and I gave up because it’s pretty low on my list of priorities right
now.

**Comment by gfb-47** [comment]
I think I found a solution, if it works I'll inform you guys.

**Reply by toster234** [reply]
:)
